samples/AspNetCore/Program.cs

```diff
@@ -4,6 +4,8 @@
 using Octokit.Webhooks;
 using Octokit.Webhooks.AspNetCore;
 
+#pragma warning disable CA1852
```

Comment:
Which types were causing this warning?

Reply:
This bug causes a false positive for the compiler-generated `Program` class: https://github.com/dotnet/roslyn-analyzers/issues/6141

Reply:
Which SDK version are you running? It should be fixed in `7.0.203`. I hit the same issue in `7.0.100`, but the fix finally got backported in this month's security update: https://github.com/octokit/webhooks.net/pull/163#issuecomment-1314248137

Reply:
I'll check when I'm back at my computer, but I'm fairly sure the global.json was making it use a preview of 7.0.300.

When I used VS 2022 17.5 the analysers were failing to load and I couldn't build, so I had to use VS 2022 17.6 instead.

Maybe `allowPrerelease: false` needs adding?

Reply:
Yep, so I get this:

```powershell
webhooks.net on  deployment_protection_rule via .NET v7.0.203
❯ dotnet --version
7.0.300-preview.23179.2
```

With `allowPrerelease: false` I get this:

```powershell
webhooks.net on  deployment_protection_rule via .NET v7.0.203
❯ dotnet --version
7.0.203
```

I guess that preview version missed getting the fix or something.

Reply:
I've added the setting to get it to not consume prerelease SDKs (it'll still use it if the version is explicitly set to one) and reverted this change.